src/components/AssetsTable/displayMessages.jsx

```diff
@@ -0,0 +1,161 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  assetsTablePreviewLabel: {
+    id: 'assetsTablePreviewLabel',
+    defaultMessage: 'Image Preview',
+    description: 'Label for column header',
+  },
+  assetsTableNameLabel: {
+    id: 'assetsTableNameLabel',
+    defaultMessage: 'Name',
+    description: 'Label for column header',
+  },
+  assetsTableTypeLable: {
+    id: 'assetsTableTypeLable',
+    defaultMessage: 'Type',
+    description: 'Label for column header',
+  },
+  assetsTableDateLabel: {
+    id: 'assetsTableDateLabel',
+    defaultMessage: 'Date Added',
+    description: 'Label for column header',
+  },
+  assetsTableCopyLabel: {
+    id: 'assetsTableCopyLabel',
+    defaultMessage: 'Copy URLs',
+    description: 'Label for column header',
+  },
+  assetsTableDeleteLabel: {
+    id: 'assetsTableDeleteLabel',
+    defaultMessage: 'Delete Asset',
+    description: 'Label for column header',
+  },
+  assetsTableLockLabel: {
+    id: 'assetsTableLockLabel',
+    defaultMessage: 'Lock Asset',
+    description: 'Label for column header',
+  },
+  assetsTableNoDescription: {
+    id: 'assetsTableNoDescription',
+    defaultMessage: 'Description not available',
+    description: 'Label shown when no description is available',
+  },
+  assetsTableNoPreview: {
+    id: 'assetsTableNoPreview',
+    defaultMessage: 'Preview not available',
+    description: 'Label shown when no preview is available',
+  },
+  assetsTableLockedObject: {
+    id: 'assetsTableLockedObject',
+    defaultMessage: 'Locked {thing}',
+    description: 'States that an object has just been locked',
```

Comment:
Should we be more clear about what "locked" means? I could see it not having the same meaning across languages.

Reply:
I'm good with "the docs define this term, we assume you're read them and agree with us on that definition", but we can clarify more if we want.